pkg/scheduler/core/generic_scheduler.go

```diff
@@ -231,6 +223,14 @@ func (g *genericScheduler) Schedule(ctx context.Context, state *framework.CycleS
 			FilteredNodesStatuses: filteredNodesStatuses,
 		}
 	}
+	trace.Step("Computing predicates done")
```

Comment:
Do not need to move this, otherwise it will not be printed when `len(filteredNodes) == 0`.